charts/devtron/devtron-bom.yaml

```diff
@@ -241,6 +242,14 @@ monitoring:
       imagePullPolicy: IfNotPresent
 devtronEnterprise:
   enabled: false
+  finops:
+    enabled: false
+    costSync:
+      image: ""
```

Comment:
image?
Also, make sure that it follows the helpers.tpl standard for images that we follow